### PR TITLE
Update dashboards build script to consume distribution

### PR DESCRIPTION
**Pull request by peterzhuamazon**
Signed-off-by: Peter Zhu <zhujiaxi@amazon.com>

### Description
Update dashboards build script to consume distribution
 
### Issues Resolved
Part of #1655
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peterzhuamazon** [comment]
Note: This does not include rpm arm64 because dashboards currently cannot build that for core.
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1259

**Review by dblock — APPROVED** [review]

File: scripts/components/OpenSearch-Dashboards/build.sh [hunk 1]
```diff
@@ -13,11 +13,12 @@ function usage() {
     echo -e "-s SNAPSHOT\t[Optional] Build a snapshot, default is 'false'."
     echo -e "-p PLATFORM\t[Optional] Platform, default is 'uname -s'."
     echo -e "-a ARCHITECTURE\t[Optional] Build architecture, default is 'uname -m'."
+    echo -e "-d DISTRIBUTION\t[Optional] Distribution, default is 'tar'."
```
Comment (zelinh) [inline scripts/components/OpenSearch-Dashboards/build.sh:16]: Should the default be `tar.gz` or just `tar` that will be used as extension below?

Comment (peterzhuamazon) [reply scripts/components/OpenSearch-Dashboards/build.sh:16]: tar, as passed from the run scripts of build.sh.
https://github.com/opensearch-project/opensearch-build/blob/main/src/build_workflow/build_args.py#L19

Comment (zelinh) [reply scripts/components/OpenSearch-Dashboards/build.sh:16]: Got it. Thanks!

**Pull request merged**